guides/app-developers/receive-tokens.md

```diff
@@ -17,9 +17,47 @@ export PRIVATE_KEY=''# input your own pk
 cast send 0x274a6990dE7AaE06452cbEFa266c0C6a568F0D5B 'drop()' --private-key $PRIVATE_KEY -r $L1_RPC_URL
 ```
 
-After that you can cross the claimed `Custom Gas Token` to L2 via `entrance` or follow the instructions [here](https://github.com/ethereum-optimism/specs/discussions/140#discussioncomment-9426636).
+After that you can cross the claimed `Custom Gas Token` to L2 via [the bridge](https://bridge.beta.testnet.l2.quarkchain.io) or follow the instructions [here](https://github.com/ethereum-optimism/specs/discussions/140#discussioncomment-9426636).
 
-### Get Testnet Soul Gas Token On L2
+### Get Beta Testnet Soul Gas Token On L2
+
+```bash
+export SOUL_GAS_TOKEN=0x4200000000000000000000000000000000000800
+export L2_RPC_URL='http://5.9.87.214:8545'
+export PRIVATE_KEY=''# input your own pk
+
+cast send --value 10ether $SOUL_GAS_TOKEN 'deposit()' --private-key $PRIVATE_KEY -r $L2_RPC_URL
+```
+
+
+Then if you import `0x4200000000000000000000000000000000000800` into metamask, you'll see your balance of `Soul Gas Token`.
+
+### Faucet
+
+>Go to [https://qkc-l2-faucet.eth.sep.w3link.io](https://qkc-l2-faucet.eth.sep.w3link.io).
+
+## Alpha Testnet
```

Comment:
Do we want to keep alpha testnet?

Reply:
Removed in e1feddb176b117f2919c031631f7ee8d56afaa0e .